data/tokens.json

```diff
@@ -1499,30 +1499,6 @@
       }
     }
   },
-  "Accordion": {
-    "gtmds_helper_accordion_border_open": {
-      "value": {
-        "color": "{gtmds_color_gray-4}",
-        "width": "{gtmds_border-width-base}"
-      },
-      "type": "border"
-    },
-    "gtmds_helper_accordion_heading_border_open": {
-      "value": {
-        "color": "{gtmds_color_gray-4}",
-        "width": "{gtmds_border-width-base}"
-      },
-      "type": "border"
-    },
-    "gtmds_helper_accordion_color_background_hover": {
```

Comment:
@tsnikolov This name should be revisited as well, check the tokens in the rest of the components